api/test/model_device.py

```diff
@@ -9,23 +9,25 @@ class DeviceTests(unittest.TestCase):
 
     def setUp(self):
         self.devices = DeviceTests.repository_collection.device_repository
-        self.house1id = ObjectId()
-        self.house2id = ObjectId()
+        self.houses = DeviceTests.repository_collection.house_repository
         self.room1id = ObjectId()
+        self.user1id = ObjectId()
```

Comment:
can you make these IDs refer to actual objects (e.g. by inserting an actual user, house and room?)